CMakeLists.txt

```diff
@@ -630,6 +635,7 @@ if(CMAKE_SYSTEM_NAME STREQUAL Windows AND BUILD_SHARED_LIBS)
 else()
   install(FILES
             ${Foundation_OUTPUT_FILE}
+            ${FoundationNetworking_OUTPUT_FILE}
```

Comment:
This is insufficient - you left out the windows case - see 5 lines above.

Reply:
Is this correct for the windows case?
```
@@ -626,10 +626,12 @@ set(FoundationNetworking_OUTPUT_FILE
 if(CMAKE_SYSTEM_NAME STREQUAL Windows AND BUILD_SHARED_LIBS)
   install(FILES
             ${Foundation_OUTPUT_FILE}
+            ${FoundationNetworking_OUTPUT_FILE}
           DESTINATION
             bin)
   install(FILES
             ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_IMPORT_LIBRARY_PREFIX}Foundation${CMAKE_IMPORT_LIBRARY_SUFFIX}
+            ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_IMPORT_LIBRARY_PREFIX}FoundationNetworking${CMAKE_IMPORT_LIBRARY_SUFFIX}
           DESTINATION
             lib/${swift_dir}/${swift_os})
 else()
```

Reply:
Yes, that should be sufficient.